src/test/java/com/ericsson/esj/jenkins_manager_test/JenkinsManagerExecutionTest.java

```diff
@@ -0,0 +1,61 @@
+package com.ericsson.esj.jenkins_manager_test;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+import com.ericsson.esj.JenkinsManager;
+
+public class JenkinsManagerExecutionTest {
+
+    private static final String username = "admin";
```

Comment:
These values should be set in pom. And it should be possible to be provided from command line when running maven.

Reply:
The values should be given from a test, test properties. And there will be tasks to implement test in the future.